test/ocsp/helper/helper.go

```diff
@@ -0,0 +1,188 @@
+package helper
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+
+	"golang.org/x/crypto/ocsp"
+)
+
+var method = flag.String("method", "GET", "Method to use for fetching OCSP")
+var urlOverride = flag.String("url", "", "URL of OCSP responder to override")
+var tooSoon = flag.Int("too-soon", 76, "If NextUpdate is fewer than this many hours in future, warn.")
+var ignoreExpiredCerts = flag.Bool("ignore-expired-certs", false, "If a cert is expired, don't bother requesting OCSP.")
+var expectStatus = flag.Int("expect-status", 0, "Expect response to have this numeric status (0=good, 1=revoked)")
+
+func getIssuer(cert *x509.Certificate) (*x509.Certificate, error) {
```

Comment:
Should there be a `cert != nil` check for robustness sake? This func would nil panic if a test sent it a nil cert.